src/storage/file_io.c

```diff
@@ -7479,7 +7480,7 @@ fileio_allocate_node (FILEIO_QUEUE * queue_p, FILEIO_BACKUP_HEADER * backup_head
     {
     case FILEIO_ZIP_LZ4_METHOD:
       assert (size <= LZ4_MAX_INPUT_SIZE);
-      buf_size = LZ4_compressBound (size);
+      buf_size = cubcompress::bound < cubcompress::LZ4 > (size);
```

Comment:
c파일에서 c++ 스타일인 경우 적용 검토
```suggestion
// *INDENT-OFF*
      buf_size = cubcompress::bound < cubcompress::LZ4 > (size);
// *INDENT-ON*
```